content/20.zksync-protocol/17.zksync-airbender/00.overview.md

```diff
@@ -0,0 +1,75 @@
+---
+
+[ZKsync OS](/zksync-protocol/zksyncos/overview) and Airbender have been developed in parallel to
+support modular and scalable architecture for ZK Chains.
+
+- Airbender is a new ZK proof system.
+- ZKsync OS is a new “operating system” for ZK Chains, allowing chains to choose any Virtual Machine, such as EVM, EraVM, or WASM.
+
+## What is Airbender?
+
+Airbender is ZKsync's next-generation proof system, purpose-built to enable efficient ZK proofs of RISC-V bytecode execution.
+Built on the foundation of highly optimized STARK/FRI implementations, Airbender is designed to support ZKsync's
+long-term scaling strategy by being fast, cheap, and flexible to a wide range of use cases (without compromising security).
+
+### How It Fits with ZKsync OS
+
+ZKsync OS acts as a modular execution layer, allowing chains to run various virtual machines like EVM, EraVM, or WASM.
+Airbender serves as the proving system that validates the execution of these VMs, encoded in RISC-V bytecode.
+
+## Key Features
+
+- RISC-V 32I+M instruction set
+- AIR constraints
+- Optimized DEEP STARK/FRI proofs
+- Mersenne31 prime field
+- Blake2s + Blake3 hash function
+
+The name “Airbender” is a reference to AIR constraints (Arithmetic Intermediate Representation).
+
+## Proving Pipeline
+
+The proving process follows a structured six-stage pipeline designed for optimal performance and resource utilization:
+
+- **Stage 1 - Witness Generation**: Computes Low-Degree Extensions (LDEs) of witness data and generates trace commitments.
+This stage establishes the foundational data structures for the proof.
+
+- **Stage 2 - Lookup and Memory Argument**: Configures lookup tables and memory arguments, establishing the infrastructure for efficient
+verification of memory operations and table lookups.
+
+- **Stage 3 - STARK Quotient Polynomial**: Computes the primary STARK quotient polynomial, which encodes the constraint
+satisfaction proof for the main circuit.
+
+- **Stage 4 - DEEP Polynomial**: Implements FRI batching optimizations to reduce the overall proof size and verification complexity.
+
+- **Stage 5 - FRI IOPP Generation**: Produces the final Interactive Oracle Proof of Proximity (FRI proof) that enables efficient verification.
+
+- **Stage 6 - SNARK Wrapper:** Wrap the final FRI proof into a final FFLONK proof which gets posted and verified onchain.
+
+## Get Started
+
+Get started with the RISC-V Prover tutorial:
+
+<!-- markdownlint-disable -->
+::card-group
+  ::card
+  ---
+  title: RISC-V Prover tutorial
+  icon: i-heroicons-academic-cap-solid
+  to: "https://github.com/matter-labs/zksync-airbender/blob/main/docs/tutorial.md"
+  ---
+  Get started with the RISC-V Prover tutorial
+  ::
+  ::card
+  ---
+  title: Get started with Docker
+  icon: i-heroicons-rocket-launch-solid
+  to: "https://github.com/matter-labs/zksync-airbender/tree/main/docker/anvil-bender"
+  ---
+  Get started with Docker
+  ::
+::
+<!-- markdownlint-enable -->
```

Comment:
not sure if the pages linked here are as well documented as we want to be a "Get Started" section. For example, the second link mentions "ohbender" which is a concept totally new for people reading the docs.

I'd replace this section with a less prominent list of links. And maybe include [this page](https://github.com/matter-labs/zksync-airbender/blob/main/docs/end_to_end.md) as well